ProjectPi/src/helper/Converter.java

```diff
@@ -0,0 +1,180 @@
+package helper;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class Converter {
+
+	/**
+	 * Reads the entire file at once
+	 */
+	public static byte[] fileToBytes(File file) {
+		return fileToBytes(file, (int) file.length(), false, false);
+	}
+
+	/**
+	 * Convert file to array of bytes in which each block of "header" + mtu bytes
+	 * represents a packet
+	 */
+	public static byte[] fileToPacketByteArray(File file) {
+		return fileToBytes(file, Transfer.getMTU(), true, true);
+	}
+
+	/**
+	 * Convert file to array of bytes
+	 */
+	public static byte[] fileToBytes(File file, int readSize, boolean withSeqNr, boolean withChecksum) {
```

Comment:
In de Convert class staan heel veel static methods die prima in een class hadden gekunt. Je had bijvoorbeeld een File-class kunnen hebben, en die een `toBytes` method, and `toPacketByteArray` etc.